src/plugins/workspace/public/plugin.ts

```diff
@@ -113,14 +115,50 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
     this.managementCurrentWorkspaceIdSubscription = currentWorkspaceId$.subscribe(
       (currentWorkspaceId) => {
         if (currentWorkspaceId) {
-          ['settings', 'dataSources'].forEach((appId) =>
+          ['settings', 'dataSourceManagement'].forEach((appId) =>
             management.sections.section.opensearchDashboards.getApp(appId)?.disable()
           );
         }
       }
     );
   }
 
+  /**
+   * Add workspace overview page to breadCrumb
+   * @param core CoreStart
+   * @private
+   */
+  private addWorkspaceToBreadCrumb(core: CoreStart) {
+    this.chromeSubscription = combineLatest([
+      core.workspaces.currentWorkspace$,
+      core.chrome.getBreadcrumbs$(),
+    ]).subscribe(([currentWorkspace, breadCrumbs]) => {
+      if (currentWorkspace && breadCrumbs && breadCrumbs.length > 0) {
+        const workspaceNotInBreadcrumbs =
+          breadCrumbs.findIndex((breadcrumb: ChromeBreadcrumb) => {
+            return breadcrumb.text === currentWorkspace.name;
+          }) === -1;
```

Comment:
```suggestion
        const workspaceNotInBreadcrumbs =
          breadCrumbs.findIndex((breadcrumb: ChromeBreadcrumb) => {
            return breadcrumb.text === currentWorkspace.name;
          }) === -1;
```
As workspace will always be the second breadcrumb, so maybe we can simply do a check on the second one to avoid potential mismatch?

Reply:
Thanks, changed to suggested way.